src/uu/tload/src/tload.rs

```diff
@@ -0,0 +1,171 @@
+// This file is part of the uutils procps package.
+//
+// For the full copyright and license information, please view the LICENSE
+// file that was distributed with this source code.
+
+use std::collections::VecDeque;
+use std::sync::{Arc, RwLock};
+use std::thread::{self, sleep};
+use std::time::Duration;
+
+use clap::{arg, crate_version, value_parser, ArgAction, ArgMatches, Command};
+use crossterm::event::{self, KeyCode, KeyEvent, KeyModifiers};
+use tui::{LegacyTui, ModernTui};
+use uucore::{error::UResult, format_usage, help_about, help_usage};
+
+const ABOUT: &str = help_about!("tload.md");
+const USAGE: &str = help_usage!("tload.md");
+
+mod tui;
+
+#[derive(Debug, Default, Clone)]
+struct SystemLoadAvg {
+    pub(crate) last_1: f32,
+    pub(crate) last_5: f32,
+    pub(crate) last_10: f32,
+}
+
+impl SystemLoadAvg {
+    #[cfg(target_os = "linux")]
+    fn new() -> UResult<SystemLoadAvg> {
+        use std::fs;
+        use uucore::error::USimpleError;
+
+        let result = fs::read_to_string("/proc/loadavg")?;
+        let splited = result.split(" ").collect::<Vec<_>>();
+
+        // Helper function to keep code clean
+        fn f(s: &str) -> UResult<f32> {
+            s.parse::<f32>()
+                .map_err(|e| USimpleError::new(-1, e.to_string()))
```

Comment:
We usually use `1`, and not `-1`.
```suggestion
                .map_err(|e| USimpleError::new(1, e.to_string()))
```